assets/package.json

```diff
@@ -4,10 +4,11 @@
     "deploy": "NODE_ENV=production tailwindcss --postcss --minify -i css/app.css -o ../priv/static/assets/css/app.css",
     "format": "prettier --write .",
     "format:check": "prettier --check .",
-    "lint": "eslint . --ext .ts,.tsx"
+    "lint:check": "eslint . --ext .ts,.tsx",
+    "typing:check": "tsc --noEmit --skipLibCheck"
   },
   "dependencies": {
-    "@membraneframework/membrane-webrtc-js": "^0.3.1",
+    "@jellyfish-dev/membrane-webrtc-js": "^0.4.0",
```

Comment:
This change doesn't belong to `Adding frontend check` PR. You can keep it, but please rename the commit changing that `Typing refactor` -> `Update membrane-webrtc-js` and merge without squashing

Reply:
OK, I'll change the commit message